src/components/Dropdown/Dropdown.js

```diff
@@ -27,11 +27,11 @@ const Dropdown = ({ buttonText, list, onSelect, selectedItem, theme }) => {
             `}
         >
             <div className={styles.clickArea} onClick={toggleDropdown}>
-                <span>{buttonText}</span>
+                <span>{buttonTxt}</span>
```

Comment:
Why are you trying to shorten word "Text" to "Txt" it does not bring any value! I do understand what you mean by Txt but there is no reason for shortening a word by one letter and  potentially making it harder for others to read & understand the code.